### DISCUSSION TITLE
zh/nix-store/host-your-own-binary-cache-server

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#170] [Announcements]
# zh/nix-store/host-your-own-binary-cache-server

一份非官方的新手指南

https://nixos-and-flakes.thiscute.world/zh/nix-store/host-your-own-binary-cache-server

**Comment by lifeich1** [comment]
纯个人使用的话attic配置比较简单，全nixos下不能复用的操作只有生成attic admin key和获取nix key

**Comment by 117503445** [comment]
typo: 蜜钥对 -> 密钥对

**Reply by ryan4yin** [reply]
Fixed
